sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -547,16 +548,35 @@ async def delete_item(
 
     @distributed_trace_async
     async def read_offer(self, **kwargs):
```

Comment:
Can we just remove these entirely from async?
Seems odd to deprecate an API that hasn't even GA'ed...
Let's just use the deprecation for sync, and async only has the new APIs.

Reply:
completely agreed, removed!